python/paddle/v2/framework/tests/test_squared_l2_distance_op.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from op_test_util import OpTestMeta
+from gradient_checker import GradientChecker, create_op
+import numpy as np
+
+
+class TestSquaredL2DistanceOp(unittest.TestCase):
+    __metaclass__ = OpTestMeta
+
+    def setUp(self):
+        self.type = 'squared_l2_distance'
+        self.inputs = {
+            'X': np.random.uniform(0.1, 1., (2, 3)).astype('float32'),
+            'Y': np.random.uniform(0.1, 1., (2, 3)).astype('float32')
+        }
+        subRes = self.inputs['X'] - self.inputs['Y']
+        output = subRes * subRes
+        self.outputs = {
+            'sub_result': subRes,
+            'Out': np.expand_dims(output.sum(1), 1)
+        }
+
+
+class TestSquaredL2DistanceGradOp(GradientChecker):
+    def test_squared_l2_distance(self):
+        op = create_op("squared_l2_distance")
+        inputs = {
+            'X': np.random.uniform(0.1, 1., (2, 3)).astype('float32'),
+            'Y': np.random.uniform(0.1, 1., (2, 3)).astype('float32')
+        }
+        self.check_grad(op, inputs, set(["X", "Y"]), "Out")
```

Comment:
Add `compare_grad` to check the consistency of results for GPU and CPU calculation.